src/lib/components/Swap/Toolbar/Caption.tsx

```diff
@@ -53,7 +53,7 @@ export function Trade({ trade }: { trade: InterfaceTrade<Currency, Currency, Tra
     const [a, b] = flip ? [outputAmount, inputAmount] : [inputAmount, outputAmount]
     const priceString = (!flip ? executionPrice : executionPrice?.invert())?.toSignificant(6)
 
-    const ratio = `1 ${a.currency.symbol} = ${priceString}} ${b.currency.symbol}`
+    const ratio = `1 ${a.currency.symbol} = ${priceString} ${b.currency.symbol}`
```

Comment:
Fix in a separate (tiny) PR if you can. No need for approval - just let it run the tests.